storage/mwm_id.hpp

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include "platform/country_file.hpp"
+
+#include "base/assert.hpp"
+#include "base/checked_cast.hpp"
+
+#include <cstdint>
+#include <map>
+#include <vector>
+
+namespace storage
+{
+using NumMwmId = std::uint16_t;
+
+class NumMwmIds final
```

Comment:
How do you plan to distinguish between two mwms with the same name but with different versions?